projects/media-viewer/src/lib/toolbar/main-toolbar/sub-toolbar/sub-toolbar.component.ts

```diff
@@ -14,12 +14,40 @@ export class SubToolbarComponent {
     public readonly toolbarEvents: ToolbarEventService
   ) {}
 
+  onClickHighlightToggle() {
+    this.toolbarEvents.highlightMode.next(true);
```

Comment:
i believe we already have a function in toolbar events that does this. so if hightlight was selected, draw is set to false.

Reply:
will fix